projects/core/src/occ/adapters/cms/occ-cms-page.adapter.ts

```diff
@@ -25,24 +28,6 @@ export class OccCmsPageAdapter implements CmsPageAdapter {
     pageContext: PageContext,
     fields?: string
   ): Observable<CmsStructureModel> {
-    // load page by Id
```

Comment:
I would dobule check if we can drop this logic, maybe with @WeizhengSap. If we plan to drop it, we should probably also drop the endpoint interface and default configuration for a `page` endponit.